app/screens/phone-auth-screen/phone-auth.tsx

```diff
@@ -343,7 +339,7 @@ export const WelcomePhoneValidationScreenDataInjected: ScreenType = ({
       navigation={navigation}
       login={login}
       loading={loading || hasToken}
-      error={error}
+      error={error?.message}
```

Comment:
If this is the error coming back from the server are we sure it will make sense for the user?